seahorn/sea_cex_base.yaml

```diff
@@ -43,8 +43,6 @@ verify_options:
   temp-dir: /tmp/verify-c-common
 # time and result stats
   horn-stats: true
-# produce counterexample harness
-  cex: /tmp/h.ll
```

Comment:
why is this removed? cex will not be produced without this option

Reply:
in verify script, I changed the behaviour of cex mode to create harness files with jobs specific naming; if every job outputs cex harness to the same file, during executable generation we might use the wrong harness file